main.py

```diff
@@ -1,7 +1,35 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    genres = ("Western", "Action", "Dramma")
+
+    actors = ("George Klooney", " Kianu Reaves", "Scarlett Keegan",
```

Comment:
why not create a list of tuples? you will not need to use split later)

Reply:
done 🟢